### DISCUSSION TITLE
Menu to switch driver in OBC

### DISCUSSION THREAD

**Discussion by jasper-buijs** [f1multiviewer/issue-tracker#23] [Feature requests] [answered]
In the right-click menus of the on-board-cameras, it would be nice to be able to select a different driver's OBC for that video player (similar to how you change the video size and audio device). That way, you can save the position of the players in your layouts, but during the race you can still switch to whoever is in a battle.

**Comment by JustJoostNL** [comment]
This would be great!

**Comment by f1multiviewer** [comment, marked as answer]
This is available in the next release!
